stl/inc/functional

```diff
@@ -1282,29 +1281,22 @@ _NODISCARD bool operator!=(nullptr_t, const function<_Fty>& _Other) noexcept {
 
 // PLACEHOLDERS
 template <int _Nx>
-struct _Ph { // placeholder
-};
+struct _Ph {}; // placeholder
 
 template <class _Tx>
-struct is_placeholder : integral_constant<int, 0> { // template to indicate that _Tx is not a placeholder
-};
+struct is_placeholder : integral_constant<int, 0> {}; // _Tx is not a placeholder
 
 template <int _Nx>
-struct is_placeholder<_Ph<_Nx>>
-    : integral_constant<int, _Nx> { // template specialization to indicate that _Ph<_Nx> is a placeholder
-};
+struct is_placeholder<_Ph<_Nx>> : integral_constant<int, _Nx> {}; // _Ph<_Nx> is a placeholder

```

Comment:
```suggestion
struct is_placeholder<_Ph<_Nx>> : integral_constant<int, _Nx> {}; // _Ph<non-zero> is a placeholder
```

Reply:
I think that would raise more questions than it answers, since we never form `_Ph<0>`. As this is pre-existing (I'm merely shortening the comment), I'd like to either leave this as-is or (if you want) drop the comment entirely.

Reply:
Hmm, I can actually add a `static_assert` (we didn't have that when the placeholders were added!) and adjust the comment.

Reply:
For the record, I agree this is much better than my suggestion.